docs/tutorials/sigstore-keyless.md

```diff
@@ -0,0 +1,281 @@
+copy the link that is printed out and paste it into your browser manually.
+
+Afterwards, you should see the `witness run` command finished in your terminal with a silent `0` exit code.
+
+
+### View the attestation data in the signed DSSE Envelope
+
+Next, you might want to view the attestation that you generated and saved to `test.json`:
+
+```
+cat test.json | jq -r .payload | base64 -d | jq
+```
+
+But this all means nothing if we can't trust it.
+
+### Download the Fulcio Root CA
+Sigstore Keyless Signing uses X.509 certificates to perform the signing of attestations. As such, we need to trust the Fulcio Certificate Authority (CA),
+which can be done by downloading the Fulcio Root CA Trust Bundle from the Fulcio API:
+
+```
+curl -s https://fulcio.sigstore.dev/api/v2/trustBundle > fulcio.pem 
+```
+
+If you `cat fulcio.pem`, you should see some JSON with some certificates inside. This is the Fulcio Root CA Trust Bundle, which includes the Fulcio Root CA certificate,
+as well as the Fulcio Intermediate CA certificate. We will need both of these.
+
+You will also need the root certificate for the FreeTSA Timestamping Authority:
+
+```
+curl -s https://freetsa.org/files/cacert.pem > freetsa.pem
+```
+
+### Create a Policy File
+Here is an example policy template:
+
+```
+cat <<EOF >> policy-template.json
+{
+  "expires": "2030-12-17T23:57:40-05:00",
+  "steps": {
+    "test": {
+      "name": "test",
+      "attestations": [
+        {
+          "type": "https://witness.dev/attestations/product/v0.1"
+        }
+      ],
+      "functionaries": [
+        {
+          "type": "root",
+          "certConstraint": {
+            "commonname": "*",
+            "dnsnames": [
+              "*"
+            ],
+            "emails": [
+              "{{EMAIL}}"
+            ],
+            "organizations": [
+              "*"
+            ],
+            "uris": [
+              "*"
+            ],
+            "roots": [
+              "{{FULCIO_KEYID}}"
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "roots": {
+    "{{FULCIO_KEYID}}": {
+      "certificate": "{{FULCIO_ROOT}}",
+      "intermediates": [
+        "{{FULCIO_INT}}"
+      ]
+    }
+  },
+  "timestampauthorities": {
+    "freetsa": {
+      "certificate": "{{FREETSA_ROOT}}"
+    }
+  }
+}
+EOF
+```
+You can save this to a file locally by copying the above code, pasting it into your terminal and pressing enter. This will create a file named `policy-template.json` in your current working directory.
+
+#### Things to Note
+There are a couple of fields to note in this template policy file:
+
+In the template we have defined a single functionary for the `test` step. This functionary is of type `root`, which means that it will be verified against an X.509 Root CA (Fulcio):
+```json
+  "functionaries": [
+    {
+      "type": "root",
+  ...
+```
```

Comment:
just noticed this. It renders fine and there are no errors in my text editor. @jkjell let me know if you think you know what it is. The ellipsis maybe?